addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -1620,6 +1620,53 @@ QUnit.test('chat window with a thread: keep scroll position in message list on f
     );
 });
 
+QUnit.test('chat window should scroll at the end after a new message is posted', async function (assert) {
+    assert.expect(1);
+
+    this.data['mail.channel'].records.push({
+        id: 20,
+        is_minimized: true,
+        state: 'open',
+        is_pinned: true,
+    });
+    for (let i = 0; i < 10; i++) {
+        this.data['mail.message'].records.push({
+            channel_ids: [20],
+            model: 'mail.channel',
+            res_id: 20,
+        });
+    }
+    await this.start({
+        env: {
+            browser: {
+                innerWidth: 1920,
+            },
+        },
+    });
+
+    // Set content of the composer of the chat window
+    await afterNextRender(() => {
+        document.querySelector(`.o_ComposerTextInput_textarea`).focus();
+        document.execCommand('insertText', false, 'WOLOLO');
+    });
+    await afterNextRender(() =>
+        triggerEvent(
+            document.querySelector('.o_ChatWindow .o_ComposerTextInput_textarea'),
+            'keydown',
+            { key: 'Enter' },
+        )
+    );
```

Comment:
Replace this by a click on the send button, it is more reliable and clean than manually triggering the keyboard event. We have another PR in progress that will change most of the existing occurrences of this.

Reply:
I m confuse, there is no send button on the chat window.

![image](https://user-images.githubusercontent.com/1810149/92108507-ce254a00-ede7-11ea-9a1a-23ab452eed16.png)

Reply:
Yup, doesn't apply on (non-mobile) chat window, since only keyboard shortcut works in this case.

Reply:
Ok now I'm confused too. Well then I guess this is fine